src/poetry_plugin_freeze/app.py

```diff
@@ -27,15 +27,25 @@
 class FreezeCommand(Command):
     name = "freeze-wheel"
 
-    options = [option("wheel-dir", None, "Sub-directory containing wheels")]
+    options = [
+        option("wheel-dir", None, "Sub-directory containing wheels", flag=False),
+        option(
+            "exclude",
+            short_name="-e",
+            description="A package name to exclude from freezing",
+            flag=False,
+            value_required=False,
+            multiple=True,
+        ),
+    ]
 
     def handle(self) -> int:
         self.line("freezing wheels")
         root_dir = self._io and self._io.input.option("directory") or Path.cwd()
 
         fridge = {}
         for project_root in project_roots(root_dir):
-            iced = IcedPoet(project_root)
+            iced = IcedPoet(project_root, self.option("wheel-dir"), self.option("exclude"))
```

Comment:
I noticed we were testing options by hitting IcedPoet but not actually using command line options here.

Of course fixing that shows up as an uncovered change. I'm going to need to add a command level test.